src/spec/src/browser.ts

```diff
@@ -454,15 +477,7 @@ export class Spec implements SpecLike<Spec> {
       return
     }
 
-    // spooky
-    const ghosts = Object.entries(this.options['git-hosts'])
-    for (const [name, template] of ghosts) {
-      if (this.#parseHostedGit(name, template)) {
-        this.type = 'git'
-        return
-      }
-    }
-
+    // Check registries before git hosts to avoid conflicts
     const regs = Object.entries(this.options.registries)
     if (!this.options.registries.npm) {
       regs.push(['npm', this.options.registry])
```

Comment:
Let's follow this same pattern here in order to support the new default registry:
```suggestion
    if (!this.options.registries.npm) {
      regs.push(['npm', this.options.registry])
    }
    if (!this.options.registries.gh) {
      regs.push(['gh', defaultRegistries.gh])
```